Functions.lua

```diff
@@ -67,6 +67,7 @@ local function FormatMessage(msg, doTimestamp)
     if doTimestamp then
         local timestring = GetTimeString()
         -- Color Code to match pChat default
+        ---@diagnostic disable-next-line: missing-parameter
```

Comment:
This seems to be VS Code specific stuff, I would not include it. It makes no sense in e.g. IntelliJ

Reply:
Yeah I don't necessarily like having a bunch of these around the addon - but since Dak is going over a bunch of stuff in vscode sorting issues I don't mind for now. Easy to remove in the future.